src/BenchmarksServer/Startup.cs

```diff
@@ -4024,6 +4029,9 @@ private static async Task UseMonoRuntimeAsync(string runtimeVersion, string outp
                     case "llvm-jit":
                         packageName = "Microsoft.NETCore.App.Runtime.Mono.LLVM.AOT.linux-x64".ToLowerInvariant();
                         break;
+                    case "llvm-aot":
```

Comment:
The two cases can be merged

Reply:
Ideally, they are supposed to use different packages. But I guess that they could be merged, since this one package fit both scenarios.